src/main/java/hudson/plugins/git/extensions/impl/SparseCheckoutPath.java

```diff
@@ -56,20 +59,41 @@ public String toString() {
         return path;
     }
 
-    private static class SparseCheckoutPathToPath implements Function<SparseCheckoutPath, String>, Serializable {
-        public String apply(@NonNull SparseCheckoutPath sparseCheckoutPath) {
-            return sparseCheckoutPath.getPath();
-        }
+    public Descriptor<SparseCheckoutPath> getDescriptor() {
```

Comment:
Please don't make white space changes in the git plugin.  The formatting is ugly and inconsistent, but changing white space makes it more difficult to read the changes and more difficult to interact with other proposed changes in the same file.

Reply:
I don't understand why the getDescriptor()  call moved outside `SparseCheckoutPathToPath`.  What makes that move necessary?

Reply:
This was not intentional. Somehow changed on reformating the code in IDE.